router.go

```diff
@@ -0,0 +1,102 @@
+package asynqmon
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/gorilla/mux"
+
+	"github.com/hibiken/asynq"
+)
+
+type RouterOptions struct {
+	RedisClient     redis.UniversalClient
+	Inspector       *asynq.Inspector
+	Middlewares     []mux.MiddlewareFunc
+	PayloadStringer PayloadStringer
+}
+
+func NewRouter(opts RouterOptions) *mux.Router {
```

Comment:
nit: Do we want to return a concrete type (*mux.Router) or should we hide the internal detail that we are using gorilla/mux by returning `http.Handler`? 

I don't think users would be calling anything on the returned object (*mux.Router) so I think returning the interface is sufficient in this case? If we go with http.Handler, we can always change the internal detail without breaking the API :)

Reply:
Done